### DISCUSSION TITLE
How to Prometheus alerts activeAt or startAt time get in Mail notification message?

### DISCUSSION THREAD

**Discussion by Ommkwn2001** [prometheus/prometheus#14733] [Q&A]
### What did you do?

I use the prometheus-server, prometheus-node-exporter , prometheus-kube-state-metrics , prometheus-alertmanager
I try to alerts mail message in the Prometheus use with "prometheus-alertmanager". But in the Alert mail message there are not define activeAt or startAt time.
When i browsing this Prometheus link and see "http://127.0.0.1:9090/api/v1/alerts" 
The result is :

`{
  "status": "success",
  "data": {
    "alerts": [
      {
        "labels": {
          "alertname": "PodCpuUsage",
          "namespace": "default",
          "pod": "prometheus-alertmanager-0",
          "severity": "critical"
        },
        "annotations": {
          "escription": "Pod prometheus-alertmanager-0 in namespace default has CPU usage below 10000 in the last 5 minutes.\n",
          "summary": "Pod prometheus-alertmanager-0 CPU usage below threshold",
          "timestamp": "Time: 2024-08-22 08:51:46.564 +0000 UTC\n"
        },
        "state": "firing",
        "activeAt": "2024-08-22T08:31:46.565522493Z",
        "value": "3.6831480399999996e+03"
      }
    ]
  }
}`

And my mail message is :

[1] Firing
Labels
alertname = PodCpuUsage
namespace = default
pod = prometheus-alertmanager-0
severity = critical
Annotations
escription = Pod prometheus-alertmanager-0 in namespace default has CPU usage below 10000 in the last 5 minutes.
summary = Pod prometheus-alertmanager-0 CPU usage below threshold
timestamp = Time: 2024-08-22 07:52:46.564 +0000 UTC
Source

My alerting-rule.yaml code is :

`alerting_rules.yml: |
    groups:
    - name: example_alert
      rules:
      - alert: PodCpuUsage
        expr: (sum(container_cpu_usage_seconds_total) by (pod, namespace, container) * 100) < 10000
        for: 5m
        labels:
          severity: critical
        annotations:
          timestamp: >
            Time: {{ with query "time()" }}{{ . | first | value | humanizeTimestamp }}{{ end }}
          summary: "Pod {{ $labels.pod }} CPU usage below threshold"
          escription: |
            Pod {{ $labels.pod }} in namespace {{ $labels.namespace }} has CPU usage below 10000 in the last 5 minutes.`

### What did you expect to see?

I want this "activeAt": "2024-08-22T08:31:46.565522493Z", time print in the mail message

But there are firing time "Time: 2024-08-22 07:52:46.564 +0000 UTC" print in my mail message.

### What did you see instead? Under which circumstances?

And i try this method:

`annotations:
  timestamp: >
     Time: {{ with query "time()" }}{{ . | first | value | humanizeTimestamp }}{{ end }}`

### System information

Windows 11

### Prometheus version

```text
prometheus version : 25.26.0
```

### Alertmanager version

```text
alertmanager version : 1.12.0
```

**Comment by machine424** [comment]
This is more of a how to question.
(question also asked on SO https://stackoverflow.com/questions/78900743/how-to-prometheus-alerts-activeat-or-startat-time-get-in-mail-notification-messa)
